packages/component-library/draft/Kaizen/Modal/Presets/ConfirmationModal.elm

```diff
@@ -242,6 +245,19 @@ footer config =
                 Nothing ->
                     buttonConfig
 
+        withOnConfirmDisabled buttonConfig =
+            case config.onConfirm of
+                Just confirmMsg ->
+                    case config.onConfirmDisabled of
```

Comment:
Could we use an `if` instead of a case?

Reply:
yeahh is there a best practice for when to use ifs vs case? I always assumed in Elm it seems to be preferred to use case, and I've just defaulted to that without looking into it more.

Reply:
case works fine but its mostly about staying consistent with the code style of Modal.